EIPS/eip-173.md

```diff
@@ -5,7 +5,8 @@ author: Nick Mudge <nick@perfectabstractions.com>, Dan Finlay <dan@danfinlay.com
 discussions-to: https://github.com/ethereum/EIPs/issues/173
 type: Standards Track
 category: ERC
-status: Draft
+status: Last Call
+review-period-end: 2020-08-10
```

Comment:
Would you mind bumping this up to 2020-09-03?  I know it isn't your fault, but this should really be 2 weeks after merged so people have enough time after automated notification goes out.